src/functions/utils/messages.ts

```diff
@@ -2,6 +2,8 @@
 export const messages = {
   bonusMessage:
     "Your total usable bonus is: {remainingTotalBudget}€\nAmount to be paid as salary: {remainingTotalPayable}€",
+  reservedBonusMessage:
```

Comment:
Do not duplicate texts. Have the sentence about reserved amount in a separate string and then combine on the calling side when necessary.